src/components/UserInfo.js

```diff
@@ -2,6 +2,7 @@ export default class UserInfo {
   constructor(nameElement, jobElement) {
     this.profileJobEl = document.querySelector(jobElement);
     this.profileNameEl = document.querySelector(nameElement);
+    this.profileAvatarEl = document.querySelector(".profile__picture_image");
```

Comment:
`".profile__picture_image"` should be passed into the constructor as well not to hardcode the selector